recipe/sitecustomize.py

```diff
@@ -0,0 +1,2 @@
+import site, sys, os
+site.addsitedir(os.path.join(sys.prefix, 'lib', 'site-packages'))
```

Comment:
```suggestion
site.addsitedir(os.path.join(sys.prefix, 'lib', 'python', 'site-packages'))
```
?

Reply:
I don't know if we need `abi3` at all. It's supposed to be a place where we can put packages in without python version involved.

Reply:
As this will also work with `abi4` packages (`abi4` is not yet spec'd but probably also happening soon), we should keep `abi3` out of the name.

I'm +1 on @isuruf code suggestion.

Reply:
Sounds good to me. Thanks!